### DISCUSSION TITLE
Set PW to autorestore only in certain events

### DISCUSSION THREAD

**Discussion by Drogenfeld** [kangyu-california/PersistentWindows#256] [Q&A] [answered]
Can I set the program to only autorestore after signing into WIndows or reactivating the system after it was locked/screens off? I find this program to be an interference with games, so that it throws them out of fullscreen or makes them crash when alt tabbing out and going back into the game. It'd be helpful to disable autorestore while in desktop use and only let it do it's thing when it really is necessary.
Alternatively, forbid it from interacting with certain programs like games, can that be done with task priorities?

**Comment by kangyu-california** [comment, marked as answer]
The conflict between full-screen gaming mode and new feature has been fixed in latest patch
[PersistentWindows5.43_0930.zip](https://github.com/kangyu-california/PersistentWindows/files/12776474/PersistentWindows5.43_0930.zip)

**Reply by Drogenfeld** [reply]
Thanks! But it's not possible to set further rules for the program, right?

**Reply by kangyu-california** [reply]
PW should not interfere with full screen gaming experience.
The rule is already in place.

If that rule does not work, user can manually pause/resume auto restore through menu.
